### ISSUE TITLE
Global constants can be duplicated in generation of LLVM

### ISSUE THREAD

**Issue by jimbxb** [pschachte/wybe#186]
For example:
```
!println("hello")
!println("hello")
```
Results in the following LLVM code:
```
declare external ccc  void @putchar(i8)    


declare external fastcc  void @"wybe.string.print_string<0>"(i64)    


@test.4 =    constant {i64, i64} { i64 5, i64 ptrtoint ([6 x i8]* @test.3 to i64) }


@test.3 =    constant [6 x i8] c"hello\00"


@test.2 =    constant {i64, i64} { i64 5, i64 ptrtoint ([6 x i8]* @test.1 to i64) }


@test.1 =    constant [6 x i8] c"hello\00"


declare external ccc  i8* @wybe_malloc(i32)    


declare external ccc  void @llvm.memcpy.p0i8.p0i8.i32(i8*, i8*, i32, i32, i1)    


define external fastcc  void @"test.<0>"()    {
entry:
  tail call fastcc  void  @"wybe.string.print_string<0>"(i64  ptrtoint (i64* getelementptr inbounds ({i64, i64}, {i64, i64}* @test.2, i32 0, i32 0) to i64))  
  tail call ccc  void  @putchar(i8  10)  
  tail call fastcc  void  @"wybe.string.print_string<0>"(i64  ptrtoint (i64* getelementptr inbounds ({i64, i64}, {i64, i64}* @test.4, i32 0, i32 0) to i64))  
  tail call ccc  void  @putchar(i8  10)  
  ret void 
}
```

Ideally, the duplicated global constants (`@test.1`-`4`) should not be duplicated. They're also in reverse (according to number), and this bothers me.

**Comment by 09milk** [comment]
did you check the executable? there was a bug causing the compiler generating different stuff for executable and .ll, I did a band-aid fix with #131, but as I described in that pull request, I still think there are some bug hiding somewhere related to this type of llvm optimization issue

**Comment by jimbxb** [comment]
Running `strings test | grep hello`, I find "hello" twice. If there's somewhere else I should be looking, Id be happy to check there too.
